### PR TITLE
feat: Partner sponsor testimonials

### LINKED ISSUE #333: Carousel of sponsors testimonials

### PR DESCRIPTION
**Pull request by siddhigate**
## Fixes Issue

This PR fixes the following issues:

closes #333 

## Changes proposed

Carousel of testimonials

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

testimonials

### CODE REVIEW

File: src/components/home/PartnerTestimonials/index.module.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+h2.h2 {
```
Comment (Chandraprakash-Darji) [inline src/components/home/PartnerTestimonials/index.module.css:1]: Styles for h2 is already defined in typography.css.
Try to remove and try it

**Review by Chandraprakash-Darji — APPROVED** [review]
Great work @siddhigate

**Comment by Parvej2001** [comment]
Wonderful work @siddhigate . Ready to merge

File: src/components/home/PartnerTestimonials/index.module.css (hunk 1 above)
Comment (Parvej2001) [reply src/components/home/PartnerTestimonials/index.module.css:1]: Yes @siddhigate this can be updated

**Review by Parvej2001 — APPROVED** [review]
Ready to merge

**Pull request merged**